tsl/test/expected/compression.out

```diff
@@ -1328,36 +1329,49 @@ SELECT histogram_bounds FROM pg_stats WHERE tablename = 'stattest' AND attname =
 ------------------
 (0 rows)
 
--- reltuples is initially -1 on PG14 before VACUUM/ANALYZE was run
-SELECT relpages, CASE WHEN reltuples > 0 THEN reltuples ELSE 0 END as reltuples FROM pg_class WHERE relname = :statchunk;
+SELECT relpages, reltuples FROM pg_class WHERE relname = :statchunk;
  relpages | reltuples 
 ----------+-----------
         0 |         0
 (1 row)
 
--- verify that corresponding compressed chunk's stats is updated as well.
--- reltuples is initially -1 on PG14 before VACUUM/ANALYZE was run
-SELECT relpages, CASE WHEN reltuples > 0 THEN reltuples ELSE 0 END as reltuples FROM pg_class WHERE relname = :'STAT_COMP_CHUNK_NAME';
+-- verify that corresponding compressed chunk table stats is updated as well.
+SELECT relpages, reltuples FROM pg_class WHERE relname = :'STAT_COMP_CHUNK_NAME';
  relpages | reltuples 
 ----------+-----------
-        0 |         0
+        1 |         1
 (1 row)
 
--- Verify that even a global analyze doesn't affect the chunk stats, changing message scope here
--- to hide WARNINGs for skipped tables
-SET client_min_messages TO ERROR;
-ANALYZE;
-SET client_min_messages TO NOTICE;
+-- Verify partial chunk stats are handled correctly when analyzing
+-- for both uncompressed and compressed chunk tables
+INSERT INTO stattest SELECT '2020/02/20 01:00'::TIMESTAMPTZ, 250 * 26 ;
+ANALYZE stattest;
 SELECT histogram_bounds FROM pg_stats WHERE tablename = :statchunk AND attname = 'c1';
-                                                       histogram_bounds                                                        
--------------------------------------------------------------------------------------------------------------------------------
- {0,250,500,750,1000,1250,1500,1750,2000,2250,2500,2750,3000,3250,3500,3750,4000,4250,4500,4750,5000,5250,5500,5750,6000,6250}
+ histogram_bounds 
+------------------
+ 
+(1 row)
+
+-- Hypertable will now see the histogram bounds since we have data in the uncompressed chunk table.
+SELECT histogram_bounds FROM pg_stats WHERE tablename = 'stattest' AND attname = 'c1';
+ histogram_bounds 
+------------------
+ 
```

Comment:
both this and chunk level histograms are empty ; is this expected?

Reply:
Its not a problem since I insert only a single tuple so histogram bounds are not generated. It definitely looks weird, I'll update the test to generate the bounds so it a little less confusing.